plugins/poll/db/migrate/20230614041219_delete_duplicate_poll_votes.rb

```diff
@@ -4,11 +4,11 @@ class DeleteDuplicatePollVotes < ActiveRecord::Migration[7.0]
   def up
     execute <<~SQL
       DELETE FROM poll_votes
-      WHERE (poll_id, user_id, updated_at) NOT IN (
-        SELECT poll_id, user_id, MAX(updated_at)
+      WHERE ctid NOT IN (
+        SELECT MIN(ctid)
         FROM poll_votes
-        GROUP BY poll_id, user_id
-      )
+        GROUP BY poll_id, user_id, poll_option_id
+      );
```

Comment:
Nitpick: this is not necessary when running single queries.

```suggestion
      )
```